packages/rspack/src/rspack.ts

```diff
@@ -124,7 +124,7 @@ function rspack(
 	options: MultiRspackOptions | RspackOptions,
 	callback?: Callback<Error, MultiStats> | Callback<Error, Stats>
 ) {
-	if (!asArray(options as any).every(i => rspackOptionsCheck(i))) {
+	if (!asArray(options).every(i => rspackOptionsCheck(i))) {
```

Comment:
```suggestion
	if (!asArray(options).every(rspackOptionsCheck)) {
```
make the code more point free

Reply:
Nope, rspackOptionsCheck curiously have overloaded signature that cannot use pointfree style.